app/layouts/DashboardLayout/InactivityDetect/InactivityDetect.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import { render, screen, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import '../../../testUtils/i18nForTests';
+import userEvent from '@testing-library/user-event';
+
+import { InactivityDetect } from './InactivityDetect.view';
+
+const selectedAccount = {
+  account: {
+    accountId: '1234',
+    firstBlockIndex: '1',
+    key_derivation_version: '1',
+    mainAddress: 'b58 code',
+    name: null,
+    nextSubaddressIndex: '1235',
+    object: 'account',
+  },
+  balanceStatus: {
+    accountBlockIndex: '1234',
+    isSynced: true,
+    localBlockIndex: '1234',
+    networkBlockIndex: '1234',
+    object: 'balance',
+    orphanedPmob: '1234',
+    pendingPmob: '1234',
+    secretedPmob: '1234',
+    spentPmob: '1234',
+    unspentPmob: '1234',
```

Comment:
Just a tip, though it doesn't affect here: numbers and strings should all be different, to make sure that the right one is displayed. Here it makes no difference because there are no tests for 1234 appearing anywhere, but it's a good practice anyway.